kythe/cxx/tools/generate_compile_commands/extract_compile_command.cc

```diff
@@ -89,7 +75,7 @@ std::string FormatCompilationCommand(const std::string& source_file,
   writer.Key("file");
   writer.String(source_file.c_str());
   writer.Key("directory");
-  writer.String(GetBuildDirectory().c_str());
+  writer.String("@BAZEL_ROOT@");
```

Comment:
Does this impact the cmake extractor? https://kythe.io/examples/#extracting-cmake-based-repositories

Reply:
No, this command is an extra action used only by the generate_compile_commands.sh script.